tap_pipedrive/tap.py

```diff
@@ -120,6 +120,11 @@ def __init__(self, config, state):
         self.config['start_date'] = pendulum.parse(self.config['start_date'])
         self.state = state
 
+        # page size configured from config, overrides the default value set in stream.py
+        page_size = int(self.config.get('page_size',500))
+        for stream in self.streams:
+            stream.limit = stream.next_start = page_size
```

Comment:
Why are we setting the `stream.next_start` while initializing the object? 
I think we should capture this value from the API endpoint response within the pagination function.

Reply:
i agree, and that is what its doing here(https://github.com/singer-io/tap-pipedrive/blob/master/tap_pipedrive/stream.py#L71-L82)

```python
    def paginate(self, response):
        payload = response.json()

        if 'additional_data' in payload and 'pagination' in payload['additional_data']:
            logger.debug('Paginate: valid response')
            pagination = payload['additional_data']['pagination']
            if 'more_items_in_collection' in pagination:
                self.more_items_in_collection = pagination['more_items_in_collection']

                if 'next_start' in pagination:
                    self.start = pagination['next_start']
```
i don't think the attribute `next_start` is necessary, but i am not too confident of removing it in the context of this P.R

Reply:
Have you run this code?

To me it looks like `self.streams` doesn't exist, so I'm not sure how you are looping over it.

```python
stream.limit = stream.next_start = page_size
```
This is also hard to read in my opinion.

```python
stream.limit = page_size
stream.next_start = page_size
```
This makes more sense to me.